esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/flow.cylc

```diff
@@ -3,11 +3,16 @@
     UTC mode = True
 
 [task parameters]
-    metric = radiation_budget
+    small = modes_of_variability, multimodel_products
```

Comment:
Would it be possible to use the word `fast` here rather than `small` (to match with the [requirements](https://github.com/ESMValGroup/ESMValTool/discussions/2723#discussioncomment-3260108)), please?

Reply:
See c9c2e2fd22c25a920b4de7c1e5bf784e04641ed0